src/main/java/tuteez/model/person/lesson/LessonManager.java

```diff
@@ -82,9 +77,7 @@ public boolean isExistingLesson(Lesson lesson) {
      */
     public Map<Person, ArrayList<Lesson>> getClashingLessons(UniquePersonList studentList, Lesson lesson) {
         Map<Person, ArrayList<Lesson>> clashingLessonMap = new HashMap<>();
-        Iterator<Person> students = studentList.iterator();
-        while (students.hasNext()) {
-            Person studentToCheck = students.next();
+        for (Person studentToCheck : studentList) {
             ArrayList<Lesson> clashedLessons = studentToCheck.getLessonsThatClash(lesson);
```

Comment:
If you want to be consistent with your naming of lessonList previously somewhere above, maybe can name as clashingLessonList?

Reply:
Makes sense!